.github/ISSUE_TEMPLATE/flaky-ci-test-issue.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Flaky CI Test Issue
+about: Report a flaky test in the CI pipeline
+title: "[Flaky Test] <test-name>"
+labels: bug, flaky-test
+assignees: ''
+
+---
+
+**Description of the Flaky Test:**
+
+- **Test Name:** [Name of the test that is flaky]
+- **Test Location:** [File and line number or test suite]
```

Comment:
permalink is better, because test position in the file may move

Reply:
Agree